runner/src/main.rs

```diff
@@ -1314,6 +1314,9 @@ fn run_cargo_fmt(mode: FormatMode) -> Step {
                             FormatMode::Fix => vec![],
                         },
                     ],
+                    &hashmap! {
+                        "RUST_LOG".to_string() => "warn".to_string(),
+                    },
```

Comment:
What does this do exactly?

Reply:
From the commit description: 

"If the user's local environment happens to have RUST_LOG=debug configured
then a `cargo fmt` step can take up to 3 minutes!"

I'll add a comment too.

Reply:
Cool, thank you :)